src/main/java/frc/robot/subsystems/CameraSubsystem.java

```diff
@@ -43,26 +46,72 @@ public double getPitch() {
     return m_bestTarget.getPitch();
   }
 
+  public double getPitch(PhotonTrackedTarget target) {
+    return target.getPitch();
+  }
+
   public double getYaw() {
     return m_bestTarget.getYaw();
   }
 
+  public double getYaw(PhotonTrackedTarget target) {
+    return target.getYaw();
+  }
+
   public double getSkew() {
     return m_bestTarget.getSkew();
   }
 
+  public double getSkew(PhotonTrackedTarget target) {
+    return target.getSkew();
+  }
+
   public double getArea() {
     return m_bestTarget.getArea();
   }
 
+  public double GetArea(PhotonTrackedTarget target) {
+    return target.getArea();
+  }
+
   public List<TargetCorner> getCorners() {
     return m_bestTarget.getDetectedCorners();
   }
 
+  public List<TargetCorner> getCorners(PhotonTrackedTarget target) {
+    return target.getDetectedCorners();
+  }
+
   public Transform3d getDistance() {
     return m_bestTarget.getBestCameraToTarget();
   }
 
+  public Transform3d getDistance(PhotonTrackedTarget target) {
+    return target.getBestCameraToTarget();
+  }
+
+  public double getDistanceToTarget() {
+    return (
+      PhotonUtils.calculateDistanceToTargetMeters(
+        PhotonVisionConstants.findCameraHeight(m_camera.getName()),
+        PhotonVisionConstants.findTargetHeight(m_bestTarget.getFiducialId()),
+        PhotonVisionConstants.KCameraPitchRadians,
+        Units.degreesToRadians(getPitch())
+      )
+    );
+  }
+
+  public double getDistanceToTarget(PhotonTrackedTarget target) {
```

Comment:
Does this compute 3D distance, or does it compute the 2D top-down (looking at the field from above) projection of the 3D distance?